parsers/BR.py

```diff
@@ -1,191 +1,184 @@
-#!/usr/bin/env python3
-
 from collections import defaultdict
+from parsers.BD import GENERATION_MAPPING
 
 import arrow
 import requests
 
 from .lib.validation import validate
 
 
-url = 'http://tr.ons.org.br/Content/GetBalancoEnergetico/null'
-
-generation_mapping = {
-                      u'nuclear': 'nuclear',
-                      u'eolica': 'wind',
-                      u'termica': 'unknown',
-                      u'solar': 'solar',
-                      'hydro': 'hydro'
-                      }
-
-regions = {
-           'BR-NE': u'nordeste',
-           'BR-N': u'norte',
-           'BR-CS': u'sudesteECentroOeste',
-           'BR-S': u'sul'
-           }
-
-region_exchanges = {
-                    'BR-CS->BR-S': "sul_sudeste",
-                    'BR-CS->BR-NE': "sudeste_nordeste",
-                    'BR-CS->BR-N': "sudeste_norteFic",
-                    'BR-N->BR-NE': "norteFic_nordeste"
-                    }
-
-
-region_exchanges_directions = {
-                    'BR-CS->BR-S': -1,
-                    'BR-CS->BR-NE': 1,
-                    'BR-CS->BR-N': 1,
-                    'BR-N->BR-NE': 1
-                    }
-
-countries_exchange = {
-    'UY': {
-        'name': u'uruguai',
-        'flow': 1
-    },
-    'AR': {
-        'name': u'argentina',
-        'flow': -1
-    },
-    'PY': {
-        'name': u'paraguai',
-        'flow': -1
-    }
+URL = "http://tr.ons.org.br/Content/GetBalancoEnergetico/null"
+SOURCE = "ons.org.br"
+
+GENERATION_MAPPINGE = {
+    "nuclear": "nuclear",
+    "eolica": "wind",
+    "termica": "unknown",
+    "solar": "solar",
+    "hydro": "hydro",
+}
+
+REGIONS = {
+    "BR-NE": "nordeste",
+    "BR-N": "norte",
+    "BR-CS": "sudesteECentroOeste",
+    "BR-S": "sul",
+}
+
+REGION_EXCHANGES = {
+    "BR-CS->BR-S": "sul_sudeste",
+    "BR-CS->BR-NE": "sudeste_nordeste",
+    "BR-CS->BR-N": "sudeste_norteFic",
+    "BR-N->BR-NE": "norteFic_nordeste",
+}
+
+
+REGION_EXCHANGES_DIRECTIONS = {
+    "BR-CS->BR-S": -1,
+    "BR-CS->BR-NE": 1,
+    "BR-CS->BR-N": 1,
+    "BR-N->BR-NE": 1,
+}
+
+COUNTRIES_EXCHANGE = {
+    "UY": {"name": "uruguai", "flow": 1},
+    "AR": {"name": "argentina", "flow": -1},
+    "PY": {"name": "paraguai", "flow": -1},
 }
 
 
 def get_data(session, logger):
```

Comment:
If you're doing refactoring, i'm not sure this function is worth keeping:
- it's literally two lines, which are a very standard use of a very standard library
- it's called only three times
- it doesn't even use the `logger` param

At the very least, let's remove the `logger` param. Can probably also inline the return statement, no need for `json_data` variable.